### PR TITLE
read from file

### PR DESCRIPTION
**Pull request by Vinglish**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +18,25 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final char FIRST_LETTER = 'w';
+
     public String[] readFromFile(String fileName) {
+        try (Stream<String> lines = Files.lines(Paths.get(fileName))) {
+            List<String> content = lines.collect(Collectors.toList());
+            Pattern p = Pattern.compile("(?i)\\b(" + FIRST_LETTER + "\\w*)\\b");
+            Matcher m = p.matcher(content.toString());
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:27]: try to solve without this classes

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +18,25 @@
...
+
+            while (m.find()) {
+                list.add(m.group().toLowerCase());
+            }
+            String[] arr = new String[list.size()];
+            list.toArray(arr);
+            Arrays.sort(arr);
+            return arr;
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:38]: here, you basically do nothing now. rethrow the Exception as a RuntimeException. see the checklist

Comment (Vinglish) [reply src/main/java/core/basesyntax/FileWork.java:38]: Done

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (Vinglish) [reply src/main/java/core/basesyntax/FileWork.java]: done.

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -8,7 +18,24 @@
...
+            String lines = Files.readString(Paths.get(fileName));
+            String[] content = lines.toLowerCase().split(" ");
+            StringBuilder words = new StringBuilder();
+
+            for (String word: content) {
+                if(word.startsWith(FIRST_LETTER)) {
+                    words.append(word.replaceAll("\\W", "")).append(" ");
+                }
+            }
+            String[] arr = words.length() > 0 ? words.toString().split(" ") : new String[0];
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:34]:
```suggestion
            String[] arr = words.toString().split(" ");
```

Comment (Vinglish) [reply src/main/java/core/basesyntax/FileWork.java:28]: Так не работает

**Pull request closed without merging**